Fix version bump script to change version in common cookbook attributes

enrico-usai opened the pull request:
### Description of changes
Version variables have been moved from attributes/default.rb of the main cookbook
to the attributes of the new added aws-parallelcluster-common cookbook.

### Tests
Tested executing `util/bump-version.sh 3.6.0 1.1.1` and checking the produced changes

demartinofra reviewed (commented):
Please check the increase in size of slurmctld.log during scale in/out operations before merging.

Commit:
Fix version bump script to change version in common cookbook attributes

Version variables have been moved from attributes/default.rb of the main cookbook
to the attributes of the new added aws-parallelcluster-common cookbook.

Signed-off-by: Enrico Usai <usai@amazon.com>
